qcodes/instrument_drivers/AlazarTech/ATS9440.py

```diff
@@ -0,0 +1,341 @@
+from .ATS import AlazarTech_ATS
+from .utils import TraceParameter
+from qcodes.utils import validators
+
+
+class AlazarTech_ATS9440(AlazarTech_ATS):
+    """
+    This class is the driver for the ATS9440 board
+    it inherits from the ATS base class
+    """
+    samples_divisor = 256 #32
```

Comment:
Please add a line line `channels = 4` here so that you tell the driver that this card has 4 channels